.travis.yml

```diff
@@ -16,65 +16,43 @@ before_install:
   - docker pull thesofproject/sof && docker tag thesofproject/sof sof
 
 env:
-  - PLATFORM=byt
-  - PLATFORM=cht
-  - PLATFORM=bdw
-  - PLATFORM=hsw
-  - PLATFORM=apl
-  - PLATFORM=skl
-  - PLATFORM=kbl
-  - PLATFORM=cnl
   - PLATFORM=sue
-  - PLATFORM=icl
   - PLATFORM=jsl
-  - PLATFORM=imx8
```

Comment:
jsl (should) and sue (probably) have qemu support too IIRC.

Reply:
OK, I didn't want to hide adding these in this unrelated and large (LoC) refactoring. Break only one thing a time ;-)

This PR makes more obvious that they were left out.